server/komodo-rest/src/main/java/org/komodo/rest/RestLink.java

```diff
@@ -100,6 +100,11 @@
         CONDITIONS(KomodoType.VDB_CONDITION),
 
         /**
+         * A link to a VDB resource
```

Comment:
Been really picky but can this be moved above IMPORTS since these are ordered by the Vdb hierarchy?